### DISCUSSION TITLE
v2.4 Broken on M1 Mac

### DISCUSSION THREAD

**Discussion by rjbunge99** [publicsamples/Oi-Grandad#35] [Q&A] [answered]
AUi generates no audio, VST3 crashes anytime i load a preset.

I installed the UltraPackPro from Gumroad before using the synth at all, maybe that's what broke it.

I must have done the install correctly as loading the presets in AU looks exactly like it should, just doesn't generate audio.

**Comment by RickTaylor79** [comment, marked as answer]
The preset pack is only compatible with [v1.1](https://github.com/publicsamples/Oi-Grandad/releases/tag/1.1.4) for the time being. I will eventually update the presets to v2 once it's in a more usable state, but there's no definite timeframe for this. You can still use the included samples of course.

Apologies, I should have made this clear on the product page, if you would like a refund let me know.

**Comment by rjbunge99** [comment]
Eh, no worries about refund. I realized some of the presets actually do work in the AU version, they just have Amp Volume & Sustain down all the way by default. But the Mod Page seems to update per preset load. So they still offer some utility. Also the samples are nice. Thanks.

**Reply by RickTaylor79** [reply]
Thanks! Yeah, they may work to some extent, but won't sound anything like they're supposed to. You can have 1.1 installed at the same time, so you might want to check that out anyway. Once I get round to making presets for v2 it will be a free update for everyone who purchased UltraPackPro.

Currently the looper section of v2 is pretty broken, I'm completely redoing this with a time-stretching mode. Once that's done I'll stop adding new features and start on the presets..

**Reply by rjbunge99** [reply]
cool. i'll keep an eye out. thanks.
